test/acceptance/framework/consul/consul_cluster.go

```diff
@@ -41,7 +41,6 @@ type Cluster interface {
 // HelmCluster implements Cluster and uses Helm
 // to create, destroy, and upgrade consul
 type HelmCluster struct {
-	cfg                config.TestConfig
```

Comment:
Was this attribute deleted just because it's unused?

Reply:
Yup!

Reply:
If you're further curious you could check out master and run golangci-lint against it to see the original. It was pretty verbose so I did not include it in PR but maybe that'd have been useful.
The github action that we're using in `consul-k8s` is kinda cool in that it flags these in the PR along with explanations.